api/src/main/java/org/pmiops/workbench/auth/ServiceAccounts.java

```diff
@@ -56,30 +30,18 @@ public String workbenchAccessToken(WorkbenchEnvironment workbenchEnvironment, Li
    *
    * <p>See docs/domain-delegation.md for more details.
    *
-   * @param serviceAccountCredential
+   * @param originalCredentials
    * @param userEmail Email address of the user to impersonate.
    * @param scopes The list of Google / OAuth API scopes to be authorized for.
    * @return
    * @throws IOException
    */
-  public GoogleCredential getImpersonatedCredential(
-      GoogleCredential serviceAccountCredential, String userEmail, List<String> scopes)
+  public static GoogleCredentials getImpersonatedCredentials(
+      GoogleCredentials originalCredentials, String userEmail, List<String> scopes)
```

Comment:
is this the `username` property (i.e. not contact email) in the new usage? If so, I recommend `username`. as a parameter name.